pandas/tests/extension/test_string.py

```diff
@@ -94,9 +94,9 @@ class TestMethods(base.BaseMethodsTests):
     def test_value_counts(self, all_data, dropna):
         return super().test_value_counts(all_data, dropna)
 
-
-class TestCasting(base.BaseCastingTests):
-    pass
+#
+# class TestCasting(base.BaseCastingTests):
+#    pass
```

Comment:
Can you re-enable this again? Then we can look into the failure (and it will need to work anyway, before we can merge this PR)

(`__eq__` is certainly implemented for StringArray, but something else might be going wrong)

Reply:
No problem. (Sorry about that, from what I could see a lack of __eq__ looked like the issue. I'll try taking another look at it.)